application/lib/model/user_model.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:application/model/user_dto.dart';
+
+class UserModel {
+  late String name;
+  late String username;
+  late String email;
+  late String password;
+  late bool isOrganizer;
+
+  UserModel({
+    required this.name,
+    required this.username,
+    required this.email,
+    required this.password,
+    required this.isOrganizer,
+  });
+
+  UserDTO toDTO(UserModel userModel) {
+    return UserDTO(
+      name: userModel.name,
```

Comment:
A bemenő objektumra nincs szükség, hogyha az alábbi módon hivatkozunk: 
`name: this.userModel.name,`.
Kérlek ezt még módosítsuk.